shared/build.gradle.kts

```diff
@@ -0,0 +1,26 @@
+apply(from = rootProject.file("buildSrc/shared.gradle.kts"))
+
+plugins {
+    id("reportstream.project-conventions")
+}
+
+
+group = "gov.cdc.prime.reportstream"
+version = "0.0.1-SNAPSHOT"
```

Comment:
What kind of code do you envision living in the `shared` project?

Reply:
I think there will be some azure storage code that will live here as well as maybe some shared enums.